packages/react-storage/src/components/StorageBrowser/context/locationActions/locationActions.tsx

```diff
@@ -26,7 +26,11 @@ export function locationActionsReducer(
 ): LocationActionsState {
   switch (action.type) {
     case 'SET_ACTION': {
-      const selected = { ...state.selected, type: action.payload };
+      const selected = {
+        ...state.selected,
+        type: action.payload,
```

Comment:
should we rename payload to type if this is how we're using it?

Reply:
Renamed `action.payload` to `action.actionType`